src/Accounts/Session.cs

```diff
@@ -161,7 +161,9 @@ public void Dispose()
         }
         string metadata = user_input.GenRawMetadata();
         string format = user_input.Get(T2IParamTypes.ImageFormat, User.Settings.FileFormat.ImageFormat);
-        image = image.ConvertTo(format, User.Settings.FileFormat.SaveMetadata ? metadata : null, User.Settings.FileFormat.DPI);
+        string selectedBitDepth = user_input.Get(T2IParamTypes.BitDepth, "8bit");
```

Comment:
Really good point. I think I just wasn't seeing the forest for the trees. The backend already did all the work for us, no point in doing it all over again. I did have to add a check on the returned image to check how many bits it was encoded with because ImageSharp will automatically encode the image with 8bits if we don't provide the bit depth variant.